### DISCUSSION TITLE
Get Alarm code via MQTT

### DISCUSSION THREAD

**Discussion by Vippis2000** [strandborg/AlsavoCtrl#27] [General]
I have build a docker image from AlsavoCtrl using stock settings and giving parameters via docker run command. Alarms are not transmitted though. How can I enable alarms?

**Comment by Vippis2000** [comment]
Like "EE03" no water flow and so on

**Comment by strandborg** [comment]
The alarm codes are stored in params 48 onwards, see README.md. There is no special handling for them on AlsavoCtrl side

**Comment by Vippis2000** [comment]
Which MQTT Topic is related to params48

![grafik](https://github.com/strandborg/AlsavoCtrl/assets/128602940/a53835fa-3f8a-426f-b4ff-d5a5bf9455eb)

**Comment by strandborg** [comment]
Ah, you're correct, they are indeed missing from the MQTT bridge. You can add them yourself in https://github.com/strandborg/AlsavoCtrl/blob/main/alsavo.py around line 65 or so. Parameters 52-54 are exposed with the names listed in the code, you can just add the alarm codes there.
